src/Microsoft.DotNet.Interactive.FSharp/FSharpKernelExtensions.fs

```diff
@@ -17,35 +17,53 @@ open XPlot.Plotly
 [<AbstractClass; Extension; Sealed>]
 type FSharpKernelExtensions private () =
     
-    static let referenceFromType = fun (typ: Type) -> sprintf "#r \"%s\"" (typ.Assembly.Location.Replace("\\", "/"))
-    static let openNamespaceOrType = fun (whatToOpen: String) -> sprintf "open %s" whatToOpen
+    static let referenceFromType (typ: Type) = sprintf "#r \"%s\"" (typ.Assembly.Location.Replace("\\", "/"))
+    static let openNamespaceOrType (whatToOpen: string) = sprintf "open %s" whatToOpen
 
     [<Extension>]
     static member UseDefaultFormatting(kernel: FSharpKernelBase) =
         let code = 
             [
                 referenceFromType typeof<IHtmlContent>
                 referenceFromType typeof<Kernel>
-                referenceFromType typeof<FSharpPocketViewTags>
+                referenceFromType typeof<FSharpKernelHelpers.IMarker>
                 referenceFromType typeof<PlotlyChart>
                 referenceFromType typeof<Formatter>
-                openNamespaceOrType typeof<IHtmlContent>.Namespace
-                openNamespaceOrType typeof<FSharpPocketViewTags>.FullName
-                openNamespaceOrType typeof<FSharpPocketViewTags>.Namespace
-                openNamespaceOrType typeof<PlotlyChart>.Namespace
+
+                openNamespaceOrType typeof<System.Console>.Namespace
+                openNamespaceOrType typeof<System.IO.File>.Namespace
+                openNamespaceOrType typeof<System.Text.StringBuilder>.Namespace
                 openNamespaceOrType typeof<Formatter>.Namespace
-            ] |> List.reduce(fun x y -> x + Environment.NewLine + y)
+            ] |> String.concat Environment.NewLine
 
         kernel.DeferCommand(SubmitCode code)
         kernel
 
     [<Extension>]
     static member UseDefaultNamespaces(kernel: FSharpKernelBase) =
-        let code = @"
-open System
-open System.Text
-open System.Threading.Tasks
-open System.Linq"
+        // F# has its own views on what namespaces are open by default in its scripting model
+        kernel
+
+    [<Extension>]
+    static member UseExtraNamespacesForTesting(kernel: FSharpKernelBase) =
```

Comment:
It looks like this should go in the test project.

Reply:
Thanks, done